todo_app/app.py

```diff
@@ -25,25 +30,73 @@ def create_app():
             format="%(asctime)s %(levelname)s %(name)s %(threadName)s: %(message)s",
         )
 
+    app_user = User()
```

Comment:
This seems to imply your app only has one user at a time, that seems a dangerous assumption to me! Instead we should be using the `current_user` object imported from flask which loads the relevant user using the `load_user` function you've defined based on the cookie in their browser identifying their user_id